docs/user_documentation.rst

```diff
@@ -219,3 +291,46 @@ The GUI workflow is dead simple:
   is needed (such as confirming dependencies to install), DevAssistant will
   ask you, so don't go get your coffee just yet.
 - After all is done, get your coffee and enjoy.
+
+Where are the assistants located?
+---------------------------------
+
+You might ask, where are DAPs installed to? And you might already know the sort answer -
+to ``~/.devassistant``. However, the long answer is little bit more complicated.
+
+First of all, there are two things defined: ``DEVASSISTANT_HOME`` and ``DEVASSISTANT_PATH``.
+Be default, ``DEVASSISTANT_HOME`` is set to ``~/.devassistant`` and ``DEVASSISTANT_PATH`` contains
+the following:
+
+- ``~/.devassistant``
+- ``/usr/local/share/devassistant``
+- ``/usr/share/devassistant``
+
+All DAPs are installed to ``DEVASSISTANT_HOME``, however, you can run assistants from all the
+locations present in ``DEVASSISTANT_PATH``. If the assistant with the same name is present in more
+of them, locations on the beginning of the list have bigger priority.
+
+Sometimes the values of ``DEVASSISTANT_HOME`` and ``DEVASSISTANT_PATH`` are different. For example
+when installing DAPs as root, ``~/.devassistant`` is no longer there and
+``/usr/local/share/devassistant`` is used as ``DEVASSISTANT_HOME`` instead. So all the users can use
+assistants installed by root. You can also redefine the values entirely by using environment
+variables. The following example will install DAP ``python`` to ``.bah``::
+
+   $ DEVASSISTANT_HOME=.bah da pkg install python
+
+However you now cannot use assistants from ``python`` DAP unless you use that environment variable
+again.
+
+Manipulating DEVASSISTANT_PATH is very similar, but directories defined in that variable are
+used together with the default ones. If you want to avoid that, define DEVASSISTANT_NO_DEFAULT_PATH
+as well, but be sure to define DEVASSISTANT_HOME then, because this will unset the default value.
```

Comment:
s/together with/in addition to/

If you want to use only the directories in `DEVASSISTANT_PATH`, define the variable `DEVASSISTANT_NO_DEFAULT_PATH`. You must then define `DEVASSISTANT_HOME` too, because its default value is unset in the process.